sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -0,0 +1,348 @@
+## API Report File for "@azure/event-hubs"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
```

Comment:
Is this warning generated or we added it?
If it is the latter, then I would change it to "Do not edit this file manually" just to add more clarity

Reply:
It's created by api-extractor.